<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrobblerSettingsPage</class>
 <widget class="QWidget" name="ScrobblerSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scrobbler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="checkbox_enable">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_scrobble_info">
     <property name="text">
      <string>Songs are scrobbled if they have valid metadata and are longer than 30 seconds, have been playing for at least half its duration or for 4 minutes (whichever occurs earlier).</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_offline">
     <property name="text">
      <string>Work in offline mode (Only cache scrobbles)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_scrobble_button">
     <property name="text">
      <string>Show scrobble button</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_submit">
     <item>
      <widget class="QLabel" name="label_submit">
       <property name="text">
        <string>Submit scrobbles every</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinbox_submit">
       <property name="suffix">
        <string> minutes</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_submit">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_submit_info">
     <property name="text">
      <string>(This is the delay between when a song is scrobbled and when scrobbles are submitted to the server. Setting the time to 0 minutes will submit scrobbles immediately).</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_lastfm">
     <property name="title">
      <string>Last.fm</string>
     </property>
     <layout class="QVBoxLayout" name="layout_lastfm">
      <item>
       <widget class="QCheckBox" name="checkbox_lastfm_enable">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_lastfm_https">
        <property name="text">
         <string>Use HTTPS for local redirectserver to bypass login problems</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LoginStateWidget" name="widget_lastfm_login_state" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_lastfm_login" native="true">
        <layout class="QVBoxLayout" name="layout_lastfm_login">
         <item>
          <layout class="QHBoxLayout" name="layout_lastfm_button_login">
           <item>
            <widget class="QPushButton" name="button_lastfm_login">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer_lastfm_login">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_librefm">
     <property name="title">
      <string>Libre.fm</string>
     </property>
     <layout class="QVBoxLayout" name="layout_librefm">
      <item>
       <widget class="QCheckBox" name="checkbox_librefm_enable">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LoginStateWidget" name="widget_librefm_login_state" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_librefm_login" native="true">
        <layout class="QVBoxLayout" name="layout_librefm_login">
         <item>
          <layout class="QHBoxLayout" name="layout_librefm_button_login">
           <item>
            <widget class="QPushButton" name="button_librefm_login">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer_librefm_login">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_listenbrainz">
     <property name="title">
      <string>Listenbrainz</string>
     </property>
     <layout class="QVBoxLayout" name="layout_listenbrainz">
      <item>
       <widget class="QCheckBox" name="checkbox_listenbrainz_enable">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_listenbrainz_user_token">
        <item>
         <widget class="QLabel" name="label_listenbrainz_user_token">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>User token:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineedit_listenbrainz_user_token"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter your user token found on: &lt;a href=&quot;https://listenbrainz.org/profile/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://listenbrainz.org/profile/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LoginStateWidget" name="widget_listenbrainz_login_state" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_listenbrainz_login" native="true">
        <layout class="QVBoxLayout" name="layout_listenbrainz_login">
         <item>
          <layout class="QHBoxLayout" name="layout_listenbrainz_button_login">
           <item>
            <widget class="QPushButton" name="button_listenbrainz_login">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer_listenbrainz_login">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginStateWidget</class>
   <extends>QWidget</extends>
   <header>widgets/loginstatewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
